src/Components/Movies/Movies.js

```diff
@@ -1,29 +1,25 @@
-import MovieCard from '../MovieCard/MovieCard'
-import './Movies.css'
+import { Link, useParams } from 'react-router-dom';
+import MovieCard from '../MovieCard/MovieCard';
+import './Movies.css';
 
-function Movies({movies, showSingleMovie}) {
-  let movieCards
-  if (!movies.movies) {
-    return
-  } else {
-    movieCards = movies.movies.map(movie => {
-    return (
-      <MovieCard 
-        key={movie.id}
-        id={movie.id}
-        title={movie.title}
-        img={movie.poster_path}
-        rating={movie.average_rating}
-        showSingleMovie={showSingleMovie}
-      />
-    )
-  })
-}
-  return (
-    <div className='movies-container'>
-      {movieCards}
-    </div>
-  )
+function Movies({ movies, showSingleMovie }) {
+  const movieCards = movies.movies
+    ? movies.movies.map((movie) => {
```

Comment:
Great job refactoring the previous 'if statement' to a turnary, looks great

Reply:
since the false option of this ternary is null, consider doing `movies.movies &&` instead of your ternary